modules/troubleshooting-network-observability-loki-tenant-rate-limit.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/troubleshooting-network-observability.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-troubleshooting-loki-tenant-rate-limit_{context}"]
+= LokiStack rate limit errors
+A rate-limit placed on the Loki tenant can result in potential temporary loss of data and a 429 error: 'Per stream rate limit exceeded (limit:xMB/sec) while attempting to ingest for stream'. You might consider having an alert set to notify you of this error. For more information, see _Creating Loki rate limit alerts for the NetObserv dashboard_ in the Additional Resources of this section.
+
+You can update the LokiStack CRD with the `perStreamRateLimit` and `perStreamRateLimitBurst` specifications, as shown in the following procedure.
+
+.Procedure
+. Navigate to *Operators* -> *Installed Operators*, viewing *All projects* from the *Project* dropdown.
+. Look for *Loki Operator*. In the details, under *Provided APIs*, select *LokiStack*.
+. Edit the YAML in the *YAML view* to add the `perStreamRateLimit` and `perStreamRateLimitBurst` specifications:
++
+[source, yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: loki
+  namespace: netobserv
+spec:
+  limits:
+    global:
+      ingestion:
+        ingestionBurstSize: 100
+        ingestionRate: 500
+        maxGlobalStreamsPerTenant: 100000
+        perStreamRateLimit: 6
+        perStreamRateLimitBurst: 30
```

Comment:
```suggestion
      ingestion:
        perStreamRateLimit: 6
        perStreamRateLimitBurst: 30
```

We should either remove the others or explain each one by one :thinking: 
https://loki-operator.dev/docs/api.md/#loki-grafana-com-v1-IngestionLimitSpec

Reply:
Ahh yea I guess those others aren't relevant even though they are part of ingestion. I can remove them.